### PR TITLE
Dev model

### PR DESCRIPTION
**Pull request by HoXuanHieu**
please update model into dev-branch

### CODE REVIEW

**Comment by th4tkh13m** [comment]
Not done yet. Should not merge.

File: src/main/java/com/example/codeE/model/user/StudentEnrollment.java [hunk 1]
```diff
@@ -2,14 +2,12 @@
 
 import java.util.Date;
 
-import lombok.AllArgsConstructor;
-import lombok.Getter;
-import lombok.NonNull;
-import lombok.Setter;
+import lombok.*;
```
Comment (th4tkh13m) [inline src/main/java/com/example/codeE/model/user/StudentEnrollment.java:5]: ?

**Review by th4tkh13m — CHANGES_REQUESTED** [review]

File: src/main/java/com/example/codeE/model/user/Submission.java [hunk 2]
```diff
@@ -20,6 +20,7 @@ public class Submission {
     @NonNull
     private String submission;
     private float score;
+    private String description ;
```
Comment (th4tkh13m) [inline src/main/java/com/example/codeE/model/user/Submission.java:23]: Can you reformat please

**Review by th4tkh13m — CHANGES_REQUESTED** [review]

File: src/main/java/com/example/codeE/model/group/Group.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+import lombok.NonNull;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class Group {
+    // groupId, groupName, courseId, dateCreate, description, numberMember   
+    //primary key
```
Comment (th4tkh13m) [inline src/main/java/com/example/codeE/model/group/Group.java:17]: Remove this

**Review by th4tkh13m — CHANGES_REQUESTED** [review]

**Comment by HoXuanHieu** [comment]
làm ơn

**Review by th4tkh13m — APPROVED** [review]
lgtm

**Pull request merged**